mscore/preferences.cpp

```diff
@@ -127,6 +127,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_IO_MIDI_ENABLEINPUT,                             new BoolPreference(true, false)},
             {PREF_IO_MIDI_EXPANDREPEATS,                           new BoolPreference(true, false)},
             {PREF_IO_MIDI_EXPORTRPNS,                              new BoolPreference(false, false)},
+            {PREF_IO_MIDI_PEDAL_EVENTS_MIN_TICKS,                  new IntPreference(1)},
```

Comment:
`IntPreference(1, true)` to (explicitly) make it available in the Advanced preferences

Reply:
I didn't write that because all the properties that has a second parameter is set to false. So, I interpreted that only when you want it to be hide in the advanced tab is when you have to use the second parameter.

Of course I do not mind to change it, and also the property name; I was not sure about how to name it.

Reply:
True, it is not needed.

Reply:
Yeah, `true` is the default parameter here